src/member.cc

```diff
@@ -1042,12 +1183,35 @@ int RedisModule_OnLoad(RedisModuleCtx* ctx,
   default:
     return REDISMODULE_ERR;
   }
-  LOG(INFO) << "GcsMode: " << module.GcsModeString();
+  RedisModule_Log(ctx,
+                  "notice",
+                  "GCS mode: %s",
+                  module.GcsModeString().c_str());
+
+  long long master_mode = 0;
```

Comment:
I accidentally duplicated this argument-parsing logic while merging. It'll be fixed when I push an update in about 3min.